tests/stockfish/test_models.py

```diff
@@ -299,3 +320,54 @@ def test_get_top_moves_raising_error(self):
             stockfish.get_top_moves(0)
         assert len(stockfish.get_top_moves(2)) == 2
         assert stockfish.get_parameters()["MultiPV"] == 1
+
+    def test_make_moves_from_current_position(self):
+        stockfish = Stockfish()
+        stockfish.set_fen_position(
+            "r1bqkb1r/pppp1ppp/2n2n2/1B2p3/4P3/5N2/PPPP1PPP/RNBQK2R w KQkq - 0 1"
+        )
+        with pytest.raises(ValueError):
+            stockfish.make_moves_from_current_position([])
+        stockfish.make_moves_from_current_position(["e1g1"])
+        assert (
+            stockfish.get_fen_position()
+            == "r1bqkb1r/pppp1ppp/2n2n2/1B2p3/4P3/5N2/PPPP1PPP/RNBQ1RK1 b kq - 1 1"
+        )
+        stockfish.make_moves_from_current_position(
+            ["f6e4", "d2d4", "e4d6", "b5c6", "d7c6", "d4e5", "d6f5"]
+        )
+        assert (
+            stockfish.get_fen_position()
+            == "r1bqkb1r/ppp2ppp/2p5/4Pn2/8/5N2/PPP2PPP/RNBQ1RK1 w kq - 1 5"
+        )
+        stockfish.make_moves_from_current_position(
+            ["d1d8", "e8d8", "b1c3", "d8e8", "f1d1", "f5e7", "h2h3", "f7f5"]
+        )
+        assert (
+            stockfish.get_fen_position()
+            == "r1b1kb1r/ppp1n1pp/2p5/4Pp2/8/2N2N1P/PPP2PP1/R1BR2K1 w - f6 0 9"
+        )
+
+    def test_make_moves_TT_speed(self):
```

Comment:
Please explain  what TT means and  what is checked in this test

Reply:
Please explain this test

Reply:
@zhelyabuzhsky Thanks for the feedback, I have updated my branch. For that test function, I've included a comment in the code describing that it does.